internal/broker/broker.go

```diff
@@ -154,6 +155,9 @@ func New(cfg Config, args ...Option) (b *Broker, err error) {
 func (b *Broker) NewSession(username, lang, mode string) (sessionID, encryptionKey string, err error) {
 	defer decorate.OnError(&err, "could not create new session for user %q", username)
 
+	// authd usernames are lowercase
+	username = strings.ToLower(username)
+
```

Comment:
I feel this is not required anymore, since if we get here, then authd has already called the broker with a lower-case username, no?

At least, an authd version that is new enough... So maybe you wanted to cover partial upgrades here?